app/components/children/Register.js

```diff
@@ -24,59 +24,77 @@ class Register extends React.Component {
-              <input for="passwordConfirmation" type="password" className="validate" 
-                value={this.state.passwordConfirmation} 
-                name="passwordConfirmation" 
-                // errorText={errors.passwordConfirmation} 
-                onChange={this.handleUserChange}
-                required />
-
-            <div className="input-field col s12">
-              <select name="userType">
-                <option defaultValue="" disabled selected>Choose your option</option>
-                <option value="employee">Employee</option>
-                <option value="manager">Manager</option>
-              </select>
-              <label>Select User Type</label>
+            <div className="row" id="loginForm">
+                <div className="col m8 offset-m2">
+                    <div className="card-panel">
+                        <div className="row grey lighten-5">
+                            <div className="col s12 center">
+                                <h4 className="blue-text text-darken-1">Register</h4>
+                            </div>
+                        </div>
+                        <form action="/register" method="POST" onSubmit={this.handleLogin}>
+                            <div className="row">
+                                <div className="col s12">
+                                    <input
+                                        placeholder="Username"
+                                        type="text"
+                                        className="validate"
+                                        value={this.state.username}
+                                        name="username"
+                                        onChange={this.handleUserChange} />
+                                </div>
+                            </div>
+                            <div className="row">
+                                <div className="col s12">
+                                    <input
+                                        placeholder="Email"
+                                        type="email"
+                                        className="validate"
+                                        value={this.state.email}
+                                        name="email"
+                                        onChange={this.handleUserChange} />
+                                </div>
+                            </div>
+                            <div className="row">
+                                <div className="col s12">
+                                    <input
+                                        placeholder="Password"
+                                        type="password"
+                                        className="validate"
+                                        value={this.state.password}
+                                        name="password"
+                                        onChange={this.handleUserChange} />
```

Comment:
@mr-attack Wondering if user validation still works on your end.   All fields need to end with required in order for the form validation to work  ```required />```  

Also dont forget to test log in (see if you are stored in mongo)